lib/services/database/database_service.dart

```diff
@@ -0,0 +1,239 @@
+import 'dart:async';
+
+import "package:collection/collection.dart";
+import 'package:mobileprism/services/database/album_data_entry.dart';
+import 'package:mobileprism/services/database/cross_table_entry.dart';
+import 'package:mobileprism/services/database/photo_data_entry.dart';
+import 'package:path/path.dart';
```

Comment:
If you only use one or a few methods/classes from a package add `show` to only import these. 
`import 'package:path/path.dart' show join;`

Reply:
I knew the feature but i thought the compile would take care of it.